packages/atomic/src/components/common/generated-answer/generated-content/generated-markdown-content.spec.ts

```diff
@@ -0,0 +1,118 @@
+import DOMPurify from 'dompurify';
+import {html} from 'lit';
+import {afterEach, describe, expect, it, vi} from 'vitest';
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {
+  type GeneratedMarkdownContentProps,
+  renderGeneratedMarkdownContent,
+} from './generated-markdown-content';
+import * as markdownUtils from './markdown-utils';
+
+vi.mock('dompurify');
```

Comment:
Spy true is necessary in browser mode https://vitest.dev/guide/mocking/modules.html#browser-mode:~:text=Browser%20Mode%20Support

```suggestion
vi.mock('dompurify', {spy: true});
```